packages/ag-charts-community/src/chart/legend.ts

```diff
@@ -225,6 +222,8 @@ export class Legend extends BaseProperties {
 
     private readonly destroyFns: Function[] = [];
 
+    private readonly proxyLegendToolbar;
```

Comment:
Nit: Could you add an explicit type here? Not sure why this is passing CI.

Reply:
Disregard, I guess this is a new TS feature to derive the field type from the constructor assigned value.